app/theme.ts

```diff
@@ -13,6 +13,14 @@ const spanishGray = '#a39b98';
 const oswald = '"Oswald", sans-serif';
 const raleway = '"Raleway", sans-serif';
 
+const resolutions = {
+  extraSmall: '0',
+  small: '600px',
+};
+
+const handheld = `(min-width: ${resolutions.extraSmall})`;
+const desktop = `(min-width: ${resolutions.small})`;
+
```

Comment:
I'm basing it off of MUI's breakpoint system as can be seen [here](https://mui.com/material-ui/customization/breakpoints/). `min-width: 0` basically works for any width up until the next breakpoint, which would be `min-width: 600px`. However, it can be adjusted up to something like 320px (a common value for small devices), so our handheld resolution would go from 320px to 599px.